tests/test_example.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-Resources is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio Resources module to create REST APIs"""
+
+
+def test_create_record(base_client):
```

Comment:
I would go with 2nd - `test_publish_record_with_files`

However, I do not think we should remove the `put` operation, since it is not *editing* the record but adding the content:

1- The first post creates the draft, but does not contain any metadata (i.e. content)
2- The content of the record is added on the `put` operation that says `Add some new metadata`. However, said metadata is not specified yet.

This is coded according to the RFC (https://codimd.web.cern.ch/w5jnQsJ3Sn6Ne5LLEQD0hg#Publish-a-new-record-with-associated-files)

I will change the comments on top of the request operations so they reflect word-by-word the RFC, to avoid confusion.

Reply:
good for 2nd.

I am not sure about good test scope either: I prefer it very scoped, but indeed flows are broader.